counterparty-core/counterpartycore/lib/messages/dispenser.py

```diff
@@ -786,21 +821,9 @@ def dispense(db, tx):
             give_quantity = dispenser["give_quantity"]
 
             if satoshirate > 0 and give_quantity > 0:
-                if (dispenser["oracle_address"] != None) and util.enabled(  # noqa: E711
```

Comment:
not sure if I understand why this code was deleted

Reply:
I did not delete but only moved the block of code which calculates the number of assets to send in a dedicated function (`get_must_give()`) which I re-use in `dispense.compose()` to check if there are enough assets left in the to dispense.